02week/pigLatin.js

```diff
@@ -9,15 +9,62 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  let lower = word.toLowerCase();
+  let fixed = lower.trim();
+  let space = fixed.indexOf(' ');
+  
+  // let vowel = Math.min(a, e, i, o, u);
+  
+  if (space != -1){
```

Comment:
You can also use split() to get an array of words in a sentence.
https://www.w3schools.com/jsref/jsref_split.asp